pkg/action/executor/rescheduler.go

```diff
@@ -149,36 +151,48 @@ func (r *ReScheduler) preActionCheck(pod *api.Pod, node *api.Node) error {
 }
 
 func (r *ReScheduler) reSchedule(pod *api.Pod, node *api.Node) (*api.Pod, error) {
+	reScheduled := false
+	var finalErr error
+	defer func() {
+		if !reScheduled {
+			glog.V(2).Infof("Move action aborted: %v.", finalErr)
+		}
+	}()
+
```

Comment:
These seem to be redundant. We may not need this defer func, `reScheduled` bool and `finalErr`. Whenever there's error below, this `reSchedule` func will return the error and its parent func `Execute` will print the error and fail the action execution:
```
func (r *ReScheduler) Execute(input *TurboActionExecutorInput) (*TurboActionExecutorOutput, error) {
        ...
        npod, err := r.reSchedule(pod, node)
	if err != nil {
		glog.Errorf("Failed to execute pod move: %v.", err)
		return &TurboActionExecutorOutput{}, err
	}
```

Reply:
Updated!